model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/remote/RemoteInfinispanKeycloakTransaction.java

```diff
@@ -254,4 +256,18 @@ public CompletionStage<?> execute(RemoteCache<K, V> cache) {
             return cache.removeAsync(key);
         }
     }
+
+    private record TombstoneOperation<K, V>() implements Operation<K, V> {
```

Comment:
can be a constant
```java
    private static final Operation<?,?> TOMBSTONE = new Operation<>() {

        @Override
        public boolean canRemove() {
            return true;
        }

        @Override
        public CompletionStage<?> execute(RemoteCache<Object, Object> cache) {
            return CompletableFutures.completedNull();
        }
    };
```

Reply:
Thanks, updated in next commit. I used `org.infinispan.commons.util.concurrent.CompletableFutures` for this.